space/helpers.py

```diff
@@ -28,6 +28,10 @@ def is_stealth_mode():
     return (not space_may_be_open()) and (not space_is_open(get_redis()))
 
 
+def should_keep(mac):
```

Comment:
Add tests ? :)

Reply:
done in 011de06fa078c3baa22f33b91e75143e31158df1